### PR TITLE
Added 'European Women In Technology' conference

### LINKED ISSUE #114: conference: Add european women in tech
Add http://www.europeanwomenintech.com/ into conferences.

Related to #6

### PR DESCRIPTION
**Pull request by moritzsternemann**
Closes #114

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -268,6 +268,10 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
   - [Ela Conf](http://elaconf.com) *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/a/a4/Flag_of_the_United_States.svg" width="30" />
 
     A safe, inclusive tech leadership conference and community for adult (18+) women (cis and trans), trans men, and genderqueer people to connect with and empower one another.
+    
+  - [European Women In Technology](http://www.europeanwomenintech.com)
+  
+    Learn from industry leaders, connect with people from all areas of tech, and maximise your potential with empowering workshops. Let’s change the industry for the better.
```
Comment (harsh183) [inline README.md]: Remove `Let’s change the industry for the better.`

Also this sounds too much like marketing text, we don't represent them, rather we just provide a description of what they do.

Comment (moritzsternemann) [reply README.md]: Alright, will do :)

Comment (harsh183) [reply README.md]: Awesome!

**Comment by harsh183** [comment]
@moritzsternemann Looks great. Merging!

**Pull request merged**